pkg/dbutil/common.go

```diff
@@ -375,7 +375,7 @@ func GetTables(ctx context.Context, db QueryExecutor, schemaName string) (tables
 		| NTEST          | BASE TABLE |
 		+----------------+------------+
 	*/
-	query := fmt.Sprintf("SHOW FULL TABLES IN `%s` WHERE Table_Type != 'VIEW';", escapeName(schemaName))
+	query := fmt.Sprintf("SHOW FULL TABLES IN `%s` WHERE Table_Type != 'VIEW' and Table_Type != 'SEQUENCE';", escapeName(schemaName))
```

Comment:
What about
```sql
	query := fmt.Sprintf("SHOW FULL TABLES IN `%s` WHERE Table_Type = 'BASE TABLE';", escapeName(schemaName))
```

Reply:
if "BASE TABLE" can include all user defined tables, i think it ok.

Reply:
@lichunzhu done

Reply:
@lichunzhu PTAL

Reply:
lance6716 PTAL